Sources/Logging/Strings/StoreKitStrings.swift

```diff
@@ -46,20 +48,20 @@ extension StoreKitStrings: CustomStringConvertible {
         case .skrequest_failed(let error):
             return "SKRequest failed: \(error.localizedDescription)"
 
-        case .skproductsrequest_failed(let error):
-            return "SKProductsRequest failed! error: \(error.localizedDescription)"
+        case .store_products_request_failed(let error):
+            return "Store products request failed! error: \(error.localizedDescription)"
 
         case .skproductsrequest_timed_out(let afterTimeInSeconds):
             return "SKProductsRequest took longer than \(afterTimeInSeconds) seconds, " +
             "cancelling request and returning an empty set. This seems to be an App Store quirk. " +
             "If this is happening to you consistently, you might want to try using a new Sandbox account. " +
             "More information: https://rev.cat/skproductsrequest-hangs"
 
-        case .skproductsrequest_finished:
+        case .store_product_request_did_finish:
             return "SKProductsRequest did finish"
```

Comment:
This can change as well:
```suggestion
            return "Store product request did finish"
```